### PR TITLE
Fix icon advance attributes properties not change

### LINKED ISSUE #22702: Toggle icon not working in create configuration Product creation Page
### Preconditions (*)
1. Magento2.3.1

### Steps to reproduce (*)
1.Catalog > Products > New Product (Configurable) > Click Create Configuration.
2. Select Attributes > Attribute Values > Bulk Images & Price > Summary.
3. Associated Products's Toggle not working properly

### Expected result (*)
On click tab arrow should be up and down.

### Actual result (*)
On click tab arrow is not changing.

### LINKED ISSUE #26022: Admin attributes Advanced Attribute Properties icon doesn't change while collapsing block
### Preconditions (*)

1.Magento 2.3.3

### Steps to reproduce (*)

1.Login to admin
2.Store -> Attributes -> products -> properties -> Advanced attribute properties
3.check the icon of  Advanced attribute properties block

### Expected result (*)

1. expected

### Actual result (*)

1. actual

### PR DESCRIPTION
**Pull request by mrtuvn**
### Description (*)

Previously icon not change after user click on fieldset to expand content
Normal behavior icon will change to indicator user content expanded. Just like product form fieldset
After this patch icon changed correctly as expected

### Fixed Issues (if relevant)

1. Fixes magento/magento2#26022: Admin attributes Advanced Attribute Properties icon doesn't change while collapsing block
2. Fixes magento/magento2#22702 Toggle icon not working in create configuration Product creation Page
 Summary grid in modal when configurable product create
Recheck
### Manual testing scenarios (*)

1. Magento 2.3.4+
2. Login admin > Store -> Attributes / Product -> Edit or create new any attribute -> Go tab properties -> Advanced attribute properties
3. Open tab advanced attribute properties

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less [hunk 1]
```diff
@@ -122,11 +122,6 @@
                 &:before {
                     content: @icon-expand-close__content;
                 }
-                &.active {
-                    &:before {
-                        content: @icon-expand-open__content;
-                    }
-                }
             }
```
Comment (mrtuvn) [inline app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less:125]: unneccesary active selector. _show or _opened already enough for css

**Comment by edenduong** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @edenduong, here is your [new Magento instance](https://pr-28032.instances.magento-community.engineering).
Admin access: [https://pr-28032.instances.magento-community.engineering/admin_efab](https://pr-28032.instances.magento-community.engineering/admin_efab)
Login: `6fe18699` Password: `f7e4c09c0f7f`
Instance will be terminated in up to 3 hours.

**Comment by edenduong** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @edenduong, here is your [new Magento instance](https://pr-28032.instances.magento-community.engineering).
Admin access: [https://pr-28032.instances.magento-community.engineering/admin_5141](https://pr-28032.instances.magento-community.engineering/admin_5141)
Login: `d27f8fa2` Password: `3adca73269c9`
Instance will be terminated in up to 3 hours.

File: app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less [hunk 2]
```diff
@@ -122,9 +122,10 @@
                 &:before {
                     content: @icon-expand-close__content;
                 }
+
                 &.active {
                     &:before {
-                        content: @icon-expand-open__content;
+                        content: @icon-expand-close__content;
```
Comment (edenduong) [inline app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less]: I think **not active** and **active** can not have the same content : "close". And this issue https://github.com/magento/magento2/issues/22702 will occur again if you edit this line from "open" to "close"

**Review by edenduong — CHANGES_REQUESTED** [review]
Hello @mrtuvn : Thank you for your contribution. This issue https://github.com/magento/magento2/issues/22702 will occur again after your PRs. And I think the not active and active collapse can not have the same content icon "close". Could you please check it again and update your Pull Request ? Thank you !

File: app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less (hunk 2 above)
Comment (mrtuvn) [reply app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less]: Hi @edenduong after see issue 22702 i just found another problem exist!. The class opened already hardcoded in html and seem not work properly as expected. If you click on title that class not change or removed
Here is reference
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/ConfigurableProduct/view/adminhtml/web/template/variations/steps/summary-grid.html#L8

Comment (mrtuvn) [reply app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less]: If not active available content open will always used
See this 
https://github.com/magento/magento2/blob/2.4-develop/app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less#L75

File: app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less [hunk 3]
```diff
@@ -122,9 +122,10 @@
                 &:before {
                     content: @icon-expand-close__content;
                 }
+
                 &.active {
                     &:before {
-                        content: @icon-expand-open__content;
```
Comment (mrtuvn) [inline app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less:127]: actually this additional is unnecessary. With class _show or opened already enough can cover this. We don't need more specific with active here
Conclusion i will preserved this for backward compatible from last change

**Comment by edenduong** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @edenduong, here is your [new Magento instance](https://pr-28032.instances.magento-community.engineering).
Admin access: [https://pr-28032.instances.magento-community.engineering/admin_5f38](https://pr-28032.instances.magento-community.engineering/admin_5f38)
Login: `7c0752ae` Password: `42baf14985ea`
Instance will be terminated in up to 3 hours.

File: app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less (hunk 3 above)
Comment (edenduong) [reply app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less:127]: Hi @mrtuvn : Thank you. I think you can remove the .active css line.

**Review by edenduong — CHANGES_REQUESTED** [review]
Thank you @mrtuvn . I think it is ok now. Could you please remove the line .active instead of change it to "closed"?

**Comment by mrtuvn** [comment]
@matento run all tests

**Comment by edenduong** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @edenduong. Thank you for your request. I'm working on Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @edenduong, here is your [new Magento instance](https://pr-28032.instances.magento-community.engineering).
Admin access: [https://pr-28032.instances.magento-community.engineering/admin_c9cf](https://pr-28032.instances.magento-community.engineering/admin_c9cf)
Login: `57ce2c95` Password: `94a01f8c6433`
Instance will be terminated in up to 3 hours.

**Review by edenduong — APPROVED** [review]
Thank you for your contribution !

**Comment by magento-engcom-team** [comment]
Hi @edenduong, thank you for the review.
[ENGCOM-7505](https://jira.corp.magento.com/browse/ENGCOM-7505) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Manual testing scenario:**
1. Login admin > Store -> Attributes / Product -> Edit or create new any attribute -> Go tab properties -> Advanced attribute properties
2. Open tab advanced attribute properties

**Before:**
Peek 2020-05-04 14-31

**After:**
Peek 2020-05-04 16-23

**Pull request merged**